.github/dependabot.yaml

```diff
@@ -17,7 +17,6 @@ updates:
     schedule:
       interval: "daily"
       time: "04:00"
-    target-branch: "main"
```

Comment:
We can use the default branch as target branch. No need to specify this anymore.